backup-and-restore.adoc

```diff
@@ -0,0 +1,130 @@
+= Backup and restore Taiga
+:toc: left
+:toclevels: 1
+:numbered:
+:source-highlighter: pygments
+:pygments-style: friendly
+
+== Recommended backup process for Taiga6 docker
+
+This is the recommended backup process, as using Taiga6 with docker is the recommended way to run Taiga in production. This tutorial assumes that you have a Taiga6 docker instance running.
+
+This tutorial has been tested with link:https://github.com/kaleidos-ventures/taiga-docker[the official Taiga6 docker image].
+
+First, create a directory on the host where you want to store your backups in and switch into it. For this example we're going to use
+
+[source,bash]
+----
+:/# mkdir taiga-backup
+:/# cd taiga-backup
+----
+
+We have to find the container ID of the database container. To get a list of all running containers including their IDs, run
+
+[source,bash]
+----
+:/# docker ps -a
+----
+
+Search for the container with "taiga-docker_taiga-db" in the name and copy the ID.
+
+Export the database contents from the database container into your backup directory on your host machine via sql dump.
+
+[source,bash]
+----
+:/# docker exec "TAIGA-DB CONTAINER ID" pg_dump -U taiga taiga > taiga-db-backup.sql
+----
+
+Everybody will get a different container ID but to give a concrete example how this should look:
+
+[source,bash]
+----
+:/# docker exec f820941cbedc pg_dump -U taiga taiga > taiga-db-backup.sql
+----
+
+Search for the container with "taiga-docker_taiga-back_1" in the image name and copy the ID. 
+
+Then tar the media files inside the taiga-back container.
+
+[source,bash]
+----
+:/# docker exec "TAIGA-BACK CONTAINER ID" tar czf taiga-media-backup.tar.gz media
+----
+
+Copy them them out of the container into the taiga-backup directory on your host.
+
+[source,bash]
+----
+:/# docker cp "TAIGA-BACK CONTAINER ID":/taiga-back/taiga-media-backup.tar.gz .
+----
+
+Anyone should only need to backup the database and the media files but exclude the static files, cause they are recollected anytime with Django collectstatic.
```

Comment:
typo "cause" > "because" (although it's correct, we try to use an english as formal and standard as possible)

Reply:
Done ✅